lib/data/data_source/remote/folders_data_source_impl.dart

```diff
@@ -15,13 +15,19 @@ class FoldersDataSourceImpl implements FoldersDataSource {
 
   final FirebaseFirestore firestore;
   final FirebaseAuth firebaseAuth;
+  String users = 'users';
 
   String? get userId => firebaseAuth.currentUser?.uid;
 
   @override
-  Future<Success> newFolder(FolderDto dto) async {
+  Future<Success> createCollection(FolderDto dto) async {
```

Comment:
createCollection -> It doesn't says much about what the function exactly does maybe `createFolder`